pkg/operator/operator/cron.go

```diff
@@ -67,13 +77,45 @@ func operatorTelemetry() error {
 			instanceType = "unknown"
 		}
 
-		instanceTypeCounts[instanceType]++
+		isSpot := false
+		if strings.Contains(strings.ToLower(node.Labels["lifecycle"]), "spot") {
+			isSpot = true
+		}
+
 		totalInstances++
+
+		instanceInfosKey := instanceType + "_ondemand"
+		if isSpot {
+			instanceInfosKey = instanceType + "_spot"
+		}
+
+		if info, ok := instanceInfos[instanceInfosKey]; ok {
+			info.Count++
+			continue
+		}
+
+		price := aws.InstanceMetadatas[*config.Cluster.Region][instanceType].Price
+		if isSpot {
+			spotPrice, err := config.AWS.SpotInstancePrice(*config.Cluster.Region, instanceType)
```

Comment:
For large clusters with a lot of spot instances, this can get pretty expensive and may cause you to get throttled by AWS if you make so many consecutive requests in such a short period of time. We should cache this information so that we only make a request per instance type.

Reply:
Good thought! It should be cached, since if we already have an `instanceInfo` for the instance type, we increment the counter and continue (line 94)

Reply:
Ah oops I see it now.